### ISSUE TITLE
whosampled.com

### ISSUE THREAD

**Issue by xxcriticxx** [X01X012013/AdBlockProtector#63]
http://www.whosampled.com/Porn-Kings/Up-to-No-Good/

![capture](https://cloud.githubusercontent.com/assets/15007183/23285283/2b66c3ba-f9fe-11e6-89b0-a923a2088125.PNG)

**Comment by X01X012013** [comment]
Fixed in 6.46
```JavaScript
if (a.domCmp(["whosampled.com"])) {
    //Lock showAdBlockerOverlay to an empty functions
    a.readOnly("showAdBlockerOverlay", function () { });
}
```

**Comment by X01X012013** [comment]
Looks like everything is good, I'll close this issue now.
